lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -845,46 +845,47 @@ llvm::Error Caffe2ModelLoader::loadWeight(const caffe2::OperatorDef &op) {
      floats: -0.028315347
      */
 
-    auto *T = new Tensor();
     for (auto &o : op.output()) {
-      tensors_[o] = T;
-    }
+      auto *T = new Tensor();
```

Comment:
Not really related to this PR --- Just wondering why we use a loop for op.output() ?  There should be only 1 output for these ops.

Reply:
the previous behavior was not correct as it looped through all outputs and only used the last one.
current code should handle both single/multiple outputs correctly if any.